perfkitbenchmarker/providers/gcp/gce_placement_group.py

```diff
@@ -122,11 +122,21 @@ def _Create(self):
         'group-placement',
         self.name,
     )
-    if (
-        self.style == CLUSTERED
-        or gcp_flags.GCE_PLACEMENT_GROUP_MAX_DISTANCE.value
-    ):
-      # Only alpha API supported for CLUSTERED and max-distance.
+    # alpha API for CLUSTERED
+    # beta API for max-distance
+    # https://cloud.google.com/sdk/gcloud/reference/alpha/compute/resource-policies/create/group-placement
+    # https://cloud.google.com/sdk/gcloud/reference/beta/compute/resource-policies/create/group-placement
+    if gcp_flags.GCE_PLACEMENT_GROUP_MAX_DISTANCE.value:
+      cmd = gcp_util.GcloudCommand(
+          self,
+          'beta',
+          'compute',
+          'resource-policies',
+          'create',
+          'group-placement',
+          self.name,
+      )
+    if self.style == CLUSTERED:
       cmd = gcp_util.GcloudCommand(
```

Comment:
Sorry I could be misunderstanding the change: if both flags are set, wouldn't this still result in using the alpha command whereas we want it to be beta? 

Also I'm curious about the effects of just switching to use the beta gcloud command and not having to deal with alpha.

Reply:
Unless I'm misunderstanding, the alpha API supports the following values for style: clustered, collocated, unspecified-collocation. 
The beta & general APIs only support values of collocated/unspecified-collocation for style.

Additionally, the max-distance flag is only supported by the alpha/beta APIs. 

I think this means that we should use the beta API if max-distance is set. However, we should switch to the alpha API if style=clustered since the alpha API is the only one which supports this setting.

Hope I could clarify my thoughts - let me know if the change is still unclear!

Reply:
Ah thanks for the clarification - that makes sense to me.